### ISSUE TITLE
Tworzenie paczki przez shipment-management - brak dodanych umów własnych

### ISSUE THREAD

**Issue by MILsoldier** [allegro/allegro-api#9527]
Chciałem przepiąć tworzenie przesyłek z zasobu parcels na shipment, z racji, że i tak musimy uruchomić allegro international (które wymaga zasobu shipment - błąd z ubezpieczeniem się pojawia).

Niestety otrzymuje: 
operacja: GetPackageCreatingStatus_Shipment data: 26.07.2024 14:18:06
{"commandId":"c7dfa3aa-6bce-4955-96c3-2eb9611dbb1f","status":"ERROR","errors":[{"code":"CREDENTIALS_ID_NOT_PROVIDED","message":"Seller didnt provide credentials for own delivery Method","details":"Seller didnt provide credentials for own delivery Method","path":"credentialsId","userMessage":"Wybrana metoda dostawy nie obsługuje tego zamówienia"}],"shipmentId":null}

lub

operacja: GetPackageCreatingStatus_Shipment data: 26.07.2024 15:17:23
{"commandId":"20a65f14-d31c-4404-b1cf-d7760e42097a","status":"ERROR","errors":[{"code":"ERROR","message":"Processing error","details":"","path":"","userMessage":"Problem z nadaniem przesyłki. Skontaktuj się z nami. Pomożemy jak najszybciej."}],"shipmentId":null}

Wiem, że chodzi o brak credentialsID lub podanie pustego. Tylko, że na koncie nie mamy dodanych umów własnych i wysyłka na zasobie parcels działa normalnie (w cenniku w ustawieniach dostaw mamy np DHL umowa własna).  Czy przejście na shipment wymaga dodania umów własnych do wszystkich form wysyłki jakie posiadamy (poza allegrowymi)? Nie da rady wysyłać jak do tej pory? 

jeśli wymaga to dodania umów własnych do każdej z form wysyłki z cennika, to czy wpłynie to na tworzenie paczek na zasobie parcels? Nie damy rady przepiąć jednocześnie wszystkich form wysyłki, a nie chcielibyśmy sobie rozwalić aktualnej wysyłki, tylko przepinać stopniowo.

**Comment by AureliuszBrussy** [comment]
Wyjaśniam to z odpowiednim zespołem. Wrócę z odpowiedzią.

**Comment by AureliuszBrussy** [comment]
Wracam z informacją - dla metod dostawy poza Allegro Standard niezbędne jest dodanie umowy własnej, więc tutaj nie uda się tego pominąć.

**Comment by MILsoldier** [comment]
Proszę jeszcze o info czy dodanie umów własnych do istniejących form wysyłki wpłynie na zasób parcel (czy wywali nam istniejące wysyłki i będziemy musieli się od razu przepinać)? Czy mimo dodania będziemy mogli tworzyć po staremu do momentu przejścia na nowy zasób (shipment)?

**Comment by AureliuszBrussy** [comment]
Nie, nie powinno mieć to wpływu na działanie wcześniejszych metod.
